Api/index.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import cors from "cors";
+import morgan from 'morgan';
+
+const hostName = 'localhost';
+const PORT = process.env.PORT || 4000;
+
+const app = express();
+
+import officeRoutes from './src/routes/officeRoutes.js';
+import partyRoutes from './src/routes/partyRoutes.js';
+import userRouter from './src/routes/userRoutes.js';
```

Comment:
Why is this one called "Router" and the other ones "Routes"? Self-review should be detailed enough to catch something like this.